freqtrade/optimize/hyperopt.py

```diff
@@ -504,66 +359,33 @@ def start(self) -> None:
         if self.has_space('buy'):
             self.analyze.populate_indicators = Hyperopt.populate_indicators  # type: ignore
         self.processed = self.tickerdata_to_dataframe(data)
+        self.exchange = None  # type: ignore
```

Comment:
why initialize `self.exchange = None` (including Type:ignore) and then never use it?

Reply:
It is initialized and used earlier. But here we want to kill the reference to the exchange to avoid pickling errors as it will not be needed after this point.

Reply:
Oh, wasn't aware that it's initialized earlier, thanks for clarifying